framework/src/outputs/Checkpoint.C

```diff
@@ -119,61 +126,48 @@ Checkpoint::directory() const
 void
 Checkpoint::outputStep(const ExecFlagType & type)
 {
-  // Output is not allowed
-  if (!_allow_output && type != EXEC_FORCED)
-    return;
-
-  // If recovering disable output of initial condition, it was already output
-  if (type == EXEC_INITIAL && _app.isRecovering())
-    return;
-
-  // store current simulation time
-  _last_output_time = _time;
-
-  // set current type
-  _current_execute_flag = type;
-
-  // Check whether we should output, then do it.
-  if (shouldOutput())
-  {
-    TIME_SECTION("outputStep", 2, "Outputting Checkpoint");
-    output();
-  }
-
-  _current_execute_flag = EXEC_NONE;
+  FileOutput::outputStep(type);
```

Comment:
Remove the override for `Checkpoint::outputStep()` as it just calls the parents method and nothing else